sds_data_manager/lambda_code/SDSCode/upload_api.py

```diff
@@ -124,3 +78,9 @@ def lambda_handler(event, context):
         }
 
     return {"statusCode": 200, "body": json.dumps(url)}
+
+
+if __name__ == "__main__":
```

Comment:
Is this intentionally included? Or just for testing?

Reply:
yes. just for testing. good catch!